geode-docs/tools_modules/gfsh/command-pages/configure.html.md.erb

```diff
@@ -27,32 +27,31 @@ Configure Portable Data eXchange for all the cache(s) in the cluster.
-| <span class="keyword parmname">\\-\\-ignore-unread-fields</span>               | Controls whether PDX ignores fields that were unread during deserialization. The default is to preserve unread fields be including their data during serialization. However, if you configure the cache to ignore unread fields then their data will be lost during serialization.You should only set this attribute to true if you know this member will only be reading cache data. In this use case you do not need to pay the cost of preserving the unread fields since you will never be reserializing PDX data. | false   |
-| <span class="keyword parmname">\\-\\-disk-store</span>                         | Named disk store where the PDX type data will be stored.                                                                                                                                                                                                                                                                                                                                                                                                                                                               | none    |
-| <span class="keyword parmname">\\-\\-auto-serializable-classes</span>          | Configures ReflectionBasedAutoSerializer as the PDX serializer for member classes. The patterns that are matched against domain class names to determine whether they should be auto-serialized. Classes are not checked for portability to non-java languages (equivalent to `check-portability=false`).                                                                                                                                                                                                              | none    |
-| <span class="keyword parmname">\\-\\-portable-auto-serializable-classes</span> | Configures ReflectionBasedAutoSerializer as the PDX serializer for member classes. The patterns that are matched against domain class names to determine whether they should be serialized. Serialization done by the PDX autoserializer will throw an exception if the object of these classes are not portable to non-Java languages (equivalent to `check-portability=true`).                                                                                                                                       | none    |
+**Parameters, configure pdx:**
 
-<span class="tablecap">Table 1. Configure PDX Parameters</span>
+| Name                                       | Description | Default |
+|--------------------------------------------|-------------|---------|
```

Comment:
If this is copied from the command help, then I am fine with it. If not, then we will need to change the code to match this.

Reply:
I don't have much experience on what `configure PDX` do. The actual help text probably needs to be looked by the person in the storage?

Reply:
Don't worry, no code changes needed - earlier work on this JIRA ticket took care of that.
This PR addresses some of the wording, just trying to make it more precise.